javascript-algorithms-and-data-structures/learn-debugging-by-building-a-TODO-app/script.js

```diff
@@ -0,0 +1,117 @@
+const form = document.getElementById("modal");
+const confirmModal = document.getElementById("confirm-modal");
+const confirmBtn = document.getElementById("confirm-btn");
+const openModalBtn = document.getElementById("open-modal-btn");
+const closeModalBtn = document.getElementById("close-modal-btn");
+const tasksContainer = document.getElementById("tasks-container");
+const textInput = document.getElementById("text-input");
+const dateInput = document.getElementById("date-input");
+const descriptionInput = document.getElementById("textarea");
+let taskData = [];
+
+openModalBtn.addEventListener("click", () => (form.style.display = "block"));
+closeModalBtn.addEventListener("click", () => confirmModal.showModal());
```

Comment:
@naomi-lgbt 

The only time I was planning on mentioning the dialog element was here when we use the `showModal()` method.